test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIncrementalismTest.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Design.IntegrationTests
+{
+    public class BuildIncrementalismTest : MSBuildIntegrationTestBase, IClassFixture<BuildServerTestFixture>
+    {
+        private readonly string _pipeName;
+
+        public BuildIncrementalismTest(BuildServerTestFixture fixture)
+        {
+            _pipeName = fixture.PipeName;
+        }
+
+        [Fact]
+        [InitializeTestProject("SimpleMvc")]
+        public async Task BuildIncremental_SimpleMvc_PersistsTargetInputFile()
+        {
+            // Arrange
+            var thumbprintLookup = new Dictionary<string, FileThumbPrint>();
+
+            // Act 1
+            var result = await DotnetMSBuild(
+                "Build",
+                $"/p:RazorCompileOnBuild=true /p:UseRazorBuildServer=true /p:_RazorBuildServerPipeName={_pipeName} /p:_RazorForceBuildServer=true");
```

Comment:
The tests that **specifically target the server** should use look like this. The tests that don't require the server shouldn't hardcode it.

Reply:
Got it. Will remove.